CONTRIBUTING.md

```diff
@@ -12,6 +12,25 @@ Make sure to install dependencies:
 npm install
 ```
 
+## Environment Variables
+
+This table provides a quick overview of the environmental setup, with detailed explanations in the corresponding sections.
+
+| Feature                          | Local Default | CI Default         | Description                                                                                                                   |
+| -------------------------------- | ------------- | ------------------ | ----------------------------------------------------------------------------------------------------------------------------- |
+| `env.INCLUDE_SLOW_TESTS` **❗️** | `false`       | `true`             | Controls inclusion of long-running tests. Overridden by setting. Details in the [Testing](#Testing) section.                  |
+| `env.CUSTOM_CHROM_PATH`          | N/A           | Windows **❗️❗️** | Path to Chrome executable. See [plugin-lighthouse/CONTRIBUTING.md](./packages/plugin-lighthouse/CONTRIBUTING.md#chrome-path). |
```

Comment:
👨‍💻 DX: It is not clear from this that the Chrome dependency is only required for using/contributing to the Lighthouse plugin, not the CLI as a whole.

Reply:
The path is needed for all plugin users. In general there are no "unreolved" problems in setup and usage. But as we saw there are. I would add a bit of text to the plugin readme to clarify the setup ad link to the trouble shooting section.

Reply:
If I install CLI and only use ESLint and coverage plugins, I should not need to worry about any Chrome path. This dependency is (I assume) tied to Lighthouse only at the moment. Am I missing something?

Reply:
> If I install CLI and only use ESLint and coverage plugins, I should not need to worry about any Chrome path.
Yes. therefore in a `CONTRIBUTING.md`
> This dependency is (I assume) tied to Lighthouse only at the moment.
Yes therefor it is inside the `package.json` of `plugin-lighthouse`

> Am I missing something?
I guess no. But what should I change now?
It is inside a contributor related file. I added lighthouse only. Hope that fixes it.

Reply:
My point was to explicitly add to the root `CONTRIBUTING` that that environment variable is only applicable when using or contributing to **Lighthouse** plugin. I unfortunately cannot see any "Lighthouse only" but I can see a link to CONTRIBUTING of Lighthouse, could be enough.